package.json

```diff
@@ -36,7 +40,7 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^5.3.0",
     "devtron": "^1.3.0",
-    "electron-builder": "5.2.1",
+    "electron-builder": "7.9.0",
```

Comment:
I'm sorry, but from my understanding, that's a deb package problem only. Do you really expect me to test every platform? I thought the PRs are there to check the code and get tests from every platform deemed important?
The artifact provided in https://circleci.com/gh/Razzeee/desktop/49#artifacts/containers/0 runs fine in windows. Is my understanding right, that `electron-builder` is used to generate every platform, even windows?